Projekty/Projekt2/Gruszkowska_Gryszkiewicz_Chylak/KM1.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Zabrakło wprowadzenia, w którym byłoby wyjaśnienie czego dane będą dotyczyć. Poniżej warto byłoby także wyjaśnić jaką strukturę mają dane, m.in. co oznaczają liczby w ramce danych. </p><p>W ramach EDA nie tylko analizujemy dane w ujęciu statystycznym, ale także chcemy zrozumieć kontekst. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/265/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>